python/tests/avro/test_reader.py

```diff
@@ -0,0 +1,162 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+# pylint: disable=W0212
+import pathlib
+
+from iceberg.avro.reader import AvroStructProtocol, DataFileReader
```

Comment:
Style nit/opinion: I like to fully import the module that's being directly tested in a test file. I feel like it helps easily identify what's being tested with much less cognitive load (just look for `reader.*`. So what do you think of making this `from iceberg.avro import reader` and then using `reader.AvroStructProtocol` and `reader.DataFileReader` down below?

Reply:
It works both for me. I have a slight preference for just having everything imported at the top because it makes the code below more concise (having to repeat the file all the time).